changelogs/unreleased/dep-update.yml

```diff
@@ -0,0 +1,3 @@
+description: Update dependency resolutions to resolve security alerts.
+change-type: patch
+destination-branches: [master, iso6, iso5]
```

Comment:
There is no need to update iso5 as it's dropped out of the support

Reply:
and more importantly not used by any of the customers

Reply:
Sorry, I missed those when using the template to make the changelog file. Will also update the changelog template so this doesn't happen anymore